sdks/python/apache_beam/transforms/util.py

```diff
@@ -221,6 +220,9 @@ def __init__(self,
     self._clock = clock
     self._data = []
     self._ignore_next_timing = False
+
+    from apache_beam.metrics import Metrics
```

Comment:
Undo this change. Transforms should be free to use metrics (at least the public metrics write API).

Reply:
Done.